### PR TITLE
Add bootstrap-bower@0.12.0.json

### PR DESCRIPTION
**Pull request by douglasduteil**
I'm not really sure if I have to use `"format": "global"`or not...

### CODE REVIEW

**Comment by douglasduteil** [comment]
@guybedford Do you have a test suite ?

**Comment by guybedford** [comment]
@douglasduteil I have considered in the past if it is worth having all registry overrides come with tests, like the tests in https://github.com/jspm/jspm-cli/tree/master/testlibs.

But as the registry grows I think this may be unmanageable.

It is certainly worth considering to avoid regressions though.

**Comment by guybedford** [comment]
Shim implies global format, so that works fine.

File: package-overrides/github/angular-ui/bootstrap-bower@0.12.0.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "angular": ">=1 <1.3.0"
```
Comment (guybedford) [inline package-overrides/github/angular-ui/bootstrap-bower@0.12.0.json]: This isn't a valid dependency form for jspm.

Comment (guybedford) [inline package-overrides/github/angular-ui/bootstrap-bower@0.12.0.json]: Either use ~, ^ or neither. Patches can be optional too.

**Comment by douglasduteil** [comment]
Hum... I took this semver from [them repo](https://github.com/angular-ui/bootstrap-bower/blob/0.12.0/bower.json#L15).
Why can't we specify those kind of range ? 
Is the registery made fo store only minimal requirement ?
In those cases what should I prevent ? minimal versions (no version below 1.x) or maximal versions (no versions above 1.2.x ) ?

**Comment by guybedford** [comment]
I would suggest setting to `~1.2` in this case.

**Comment by douglasduteil** [comment]
Alright fixed

**Comment by guybedford** [comment]
Great.

**Pull request merged**